tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py

```diff
@@ -0,0 +1,168 @@
+        Field(item="8B", name='NUM_FAMILIES', type='number', startIndex=123, endIndex=131,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="9B", name='NUM_2_PARENTS', type='number', startIndex=147, endIndex=155,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="10B", name='NUM_1_PARENTS', type='number', startIndex=171, endIndex=179,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="11B", name='NUM_NO_PARENTS', type='number', startIndex=195, endIndex=203,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="12B", name='NUM_RECIPIENTS', type='number', startIndex=219, endIndex=227,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="13B", name='NUM_ADULT_RECIPIENTS', type='number', startIndex=243, endIndex=251,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="14B", name='NUM_CHILD_RECIPIENTS', type='number', startIndex=267, endIndex=275,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="15B", name='NUM_NONCUSTODIALS', type='number', startIndex=291, endIndex=299,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="16B", name='NUM_BIRTHS', type='number', startIndex=315, endIndex=323,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="17B", name='NUM_OUTWEDLOCK_BIRTHS', type='number', startIndex=339, endIndex=347,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="18B", name='NUM_CLOSED_CASES', type='number', startIndex=363, endIndex=371,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+    ],
+)
+
+s3 = RowSchema(
+    model=TANF_T6,
+    preparsing_validators=[
+        validators.hasLength(379),
+    ],
+    postparsing_validators=[
+        validators.sumIsEqual("NUM_APPLICATIONS", ["NUM_APPROVED", "NUM_DENIED"]),
+        validators.sumIsEqual("NUM_FAMILIES", ["NUM_2_PARENTS", "NUM_1_PARENTS", "NUM_NO_PARENTS"]),
+        validators.sumIsEqual("NUM_RECIPIENTS", ["NUM_ADULT_RECIPIENTS", "NUM_CHILD_RECIPIENTS"]),
+    ],
+    fields=[
+        Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
+              required=True, validators=[]),
+        Field(item="3A", name='CALENDAR_YEAR', type='number', startIndex=2, endIndex=6,
+              required=True, validators=[validators.month_year_yearIsLargerThan(1998)]),
+        Field(item="3B", name='CALENDAR_QUARTER', type='number', startIndex=6, endIndex=7,
+              required=True, validators=[validators.isInLimits(1, 4)]),
+        Field(item="4C", name='NUM_APPLICATIONS', type='number', startIndex=23, endIndex=31,
```

Comment:
just a question about the item numbers - we use the `A`, `B`, etc. to denote sub-fields in other schemas, here it's used to denote which record of the row is being processed (corresponding to the "first month", "second month", etc. in the instructions). i thought this was fine and made sense until i noticed `CALENDAR_YEAR` and `CALENDAR_QUARTER` above use it as a sub-field indicator. is it confusing to mix them up? could we use a different month indicator?

Reply:
i was thinking more along the lines of `4-1` in the first schema, `4-2` in the second, and so on. 3A/3B makes sense to me still and that's the way we use it in other schemas